packages/vscode-extension/src/project/project.ts

```diff
@@ -717,6 +717,13 @@ export class Project
 
   //#region Select device
   private async selectDeviceOnly(deviceInfo: DeviceInfo) {
+    if (!deviceInfo.available) {
+      window.showErrorMessage(
+        "Selected device is not available. Perhaps the device runtime is not installed. Please select another device.",
```

Comment:
In "Manage Devices" we call it the "System Image", not "Device Runtime". I think it'd be best to have the name be consistent throughout the UI.

```suggestion
        "Selected device is not available. Perhaps the system image it uses not installed. Please select another device.",
```